STYLEGUIDE.md

```diff
@@ -31,3 +33,20 @@
 - Run the linter and follow all rules defined in .eslintrc
 - Never use absolute paths in code. The app should be able to run behind a proxy under an arbitrary path.
 - TESTS: Should follow a similar "test tables" convention as used in Go where applicable.
+
+### Additional OpenShift Console Specific Practices
+
+#### React Component Patterns
+- Use `React.FCC` instead of `React.FC` for components (fixes an issue with implicit 'children' in React.FC)
```

Comment:
to be honest I don't think we should be enforcing this everywhere as when we update to react 18 next release I will have to undo all the React.FCC changes back to React.FC. We should be explicitly adding `children` prop and writing components in FCC, but I disagree with this being in STYLEGUIDE